File: 프로그래머스/2/70129. 이진 변환 반복하기/이진 변환 반복하기.js
function solution(s) {
    let count = 0;
    let zeroCount = 0;
    while(s.length !== 1){
        let zero = s.match(/0/g)
        zeroCount += (zero !== null) ? zero.length : 0
        s = s.replace(/0/g,"")
        s = Number(s.length).toString(2)
        count++
    }
    return [count, zeroCount];
}

File: 프로그래머스/2/70129. 이진 변환 반복하기/이진 변환 반복하기.cs
using System;
using System.Linq;

public class Solution {
    public int[] solution(string s) {
        
        int count = 0;
        int zeroCount = 0;
        
        while(s.Length != 1)
        {
            int oneCount = s.Where(element=> element == '1').Count();
            zeroCount += s.Length - oneCount;
            s = Convert.ToString(oneCount, 2);
            count++;
        }
        
        return new int[] { count , zeroCount };
    }
}